Automattic/jetpack
# Dashboard: update CTAs (wording, layout) in the AAG section.

#### Changes proposed in this Pull Request:

* Change the order of the buttons in the Stats section.
* Remove periods at the end of the "Learn More" links.

**Before**

<img width="581" alt="Screenshot 2022-01-13 at 19 18 56" src="https://user-images.githubusercontent.com/426388/149387041-ac7df484-9604-44b3-843f-93ec2ebd0e84.png">
<img width="707" alt="Screenshot 2022-01-13 at 19 18 39" src="https://user-images.githubusercontent.com/426388/149387047-7528b6a8-324d-4cc4-8310-df3cd9a78d6e.png">
<img width="572" alt="Screenshot 2022-01-13 at 19 18 34" src="https://user-images.githubusercontent.com/426388/149387049-5d320ea6-a558-4b13-bf9a-4edfcbaad7f4.png">

**After**

<img width="556" alt="Screenshot 2022-01-13 at 19 22 12" src="https://user-images.githubusercontent.com/426388/149387277-75d177f1-0b79-406a-9822-9569ddf8c2ec.png">
<img width="459" alt="Screenshot 2022-01-13 at 19 22 08" src="https://user-images.githubusercontent.com/426388/149387280-9aef5469-45ea-49f6-8474-43f42b0f2f24.png">
<img width="619" alt="Screenshot 2022-01-13 at 19 21 59" src="https://user-images.githubusercontent.com/426388/149387283-4f9fa375-6c9a-4bb7-85ac-f10b338fb1ad.png">

#### Jetpack product discussion

* ﻿Internal reference: p8oabR-LV-p2#comment-5865

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Go to Jetpack > Dashboard
* Check for the changes in the screenshots above.
